src/components/app/changeColors.ts

```diff
@@ -0,0 +1,19 @@
+enum Colors {
+    Light = 'light',
+    Grape = 'grape',
+    Blue = 'blue',
+    Dark = 'dark'
+}
+type ColorButtons = keyof typeof Colors;
+
+class ChangeColor {
+    changeBackgroundColor(key: ColorButtons) {
+        const body = document.querySelector('body') as Element;
+        const color = Colors[key];
+        const classToAdd = color;
+        body.classList.remove('light', 'grape', 'blue', 'dark');
```

Comment:
```suggestion
        body.classList.remove(Colors.Light, Colors.Grape, Colors.Blue, Colors.Dark);
```
or maybe better remove step by step all Colors.

Reply:
done!